src/grid/types/grid-column.ts

```diff
@@ -0,0 +1,7 @@
+import { ComponentInstance } from '../../formio.common';
+
+export interface GridColumn {
+  label: string;
```

Comment:
@vlad-shusterman label should be optional, people may want not to change the label